localMatchApp/test/tst_logintest.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <gmock/gmock-matchers.h>
+#include <gtest/gtest.h>
+#include <iostream>
+#include <string>
+#include "Login.h"
+using namespace testing;
+
+TEST ( LoginTest, TestPassword ) {
```

Comment:
Provide a description for this test.
Also check negative cases like a password not containing a digit, lower or capital letter.